doc/proposals/image-scaffold-feature.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Image scaffold feature
+authors:
+  - "@camilamacedo86"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2019-12-05
+last-updated: 2019-12-05
+status: implementable
+see-also:
+  - "/proposals/this-other-neat-thing.md"  
+replaces:
+  - "/proposals/that-less-than-great-idea.md"
+superseded-by:
+  - "/proposals/our-past-effort.md"
+---
+
```

Comment:
Hi @dmesser,

The motivation of this proposal came from the community, it would make the things very easier for the simple cases where the goal is to do an operator to install/config one image. WDYT over this idea? Have you any input?